templates/ntp.conf.j2

```diff
@@ -22,12 +25,14 @@ server {{ item }}
 # Permit time synchronization with our time source, but do not
 # permit the source to query or modify the service on this system.
 restrict default nomodify notrap nopeer noquery
+restrict -6 default nomodify notrap nopeer noquery
```

Comment:
Will these ipv6 defaults work with all OSes, or be ignored if v6 is not present/active? I don't want to accidentally break a bunch of people's servers running older software.